ccm_web/client/src/components/GradebookCanvasValidators.ts

```diff
@@ -25,11 +46,15 @@ class CurrentAndFinalGradeMatchGradebookValidator extends GradebookValidator {
   validate = (record: GradebookRecord, rowNumber: number): GradebookRowInvalidation[] => {
     const invalidations: GradebookRowInvalidation[] = []
     if (record['Final Grade'] !== record['Current Grade']) {
-      invalidations.push(new CurrentAndFinalGradeMismatchInvalidation(record, rowNumber))
+      if (record['Override Grade'] === undefined) {
+        invalidations.push(new CurrentAndFinalGradeMismatchError(record, rowNumber))
+      } else {
+        invalidations.push(new CurrentAndFinalGradeMismatchWarning(record, rowNumber))
```

Comment:
You need to send ERROR or WARNING here otherwise you see the below error
```
ERROR in /base/client/src/components/GradebookCanvasValidators.ts
ccm_web     | ./client/src/components/GradebookCanvasValidators.ts 50:27-83
ccm_web     | [tsl] ERROR in /base/client/src/components/GradebookCanvasValidators.ts(50,28)
ccm_web     |       TS2554: Expected 3 arguments, but got 2.
ccm_web     |  @ ./client/src/pages/GradebookCanvas.tsx 23:0-131 171:41-84 175:66-99 179:91-126
ccm_web     |  @ ./client/src/models/FeatureUIData.tsx 9:0-62 21:15-37
ccm_web     |  @ ./client/src/App.tsx 19:0-49 33:19-34
ccm_web     |  @ ./client/src/index.tsx 4:0-24 8:56-59
ccm_web     |
ccm_web     | ERROR in /base/client/src/components/GradebookCanvasValidators.ts
ccm_web     | ./client/src/components/GradebookCanvasValidators.ts 52:27-85
ccm_web     | [tsl] ERROR in /base/client/src/components/GradebookCanvasValidators.ts(52,28)
ccm_web     |       TS2554: Expected 3 arguments, but got 2.
ccm_web     |  @ ./client/src/pages/GradebookCanvas.tsx 23:0-131 171:41-84 175:66-99 179:91-126
ccm_web     |  @ ./client/src/models/FeatureUIData.tsx 9:0-62 21:15-37
ccm_web     |  @ ./client/src/App.tsx 19:0-49 33:19-34
ccm_web     |  @ ./client/src/index.tsx 4:0-24 8:56-59
```

Reply:
My bad I'm not sure how i missed that, it's fixed